addons/mass_mailing_event/models/event.py

```diff
@@ -20,7 +20,7 @@ def action_mass_mailing_attendees(self):
             target='current',
             context=dict(
                 default_mailing_model_id=self.env.ref('event.model_event_registration').id,
-                default_mailing_domain=domain,
+                default_mailing_domain=repr(domain),
```

Comment:
`repr` on a string?

Reply:
Then, based on the behavior of repr, could be `repr([('event_id', 'in', self.ids)])` all the time

Reply:
Did not see it was a string. Updated.